lib/msal-browser/FAQ.md

```diff
@@ -43,6 +43,8 @@
 1. [I logged out of my application. Why am I not asked for credentials when I try to log back in?](#i-logged-out-of-my-application-why-am-i-not-asked-for-credentials-when-i-try-to-log-back-in)
 1. [Why am I not signed in when returning from an invite link?](#why-am-i-not-signed-in-when-returning-from-an-invite-link)
 1. [Why is there no access token returned from acquireTokenSilent?](#why-is-there-no-access-token-returned-from-acquiretokensilent)
+1. [I used to be able to get an access token when using acquireTokenSilent with clientId as scope, now I can't. Why?](#i-used-to-be-able-to-get-an-access-token-when-using-acquiretokensilent-with-clientId-as-scope-now-i-can't-why)
```

Comment:
I think this is only relevant for msal-core

Reply:
Aah, I'll double check with the 2.x sample

Reply:
What I mean is that `msal-browser` has always behaved this way so I'm not sure we need to callout a change. The doc linked in the answer was also written specifically for `msal-core` so I would prefer to leave this out of the `msal-browser` FAQ until we have a chance to write a similar doc for `msal-browser`

Reply:
got it 👍